docs/pages/guides/webhooks.mdx

```diff
@@ -0,0 +1,196 @@
+	}
+}
+```
+
+## Endpoints
+
+An endpoint is a URL whose purpose is to receive event messages and trigger a
+task based on the message payload. The only criterion for creating an endpoint
+is that it must be one that you control.
+
+## Use Case
+
+Prior to webhooks, if you wanted to update internal data based on Liveblocks'
+state, you would need to set up a cron job to poll the
+[rooms API](https://development.liveblocks.io/docs/api-reference/rest-api-endpoints)
+for recurrent updates. With webhooks, you can now subscribe to the events you
+are interested in, and be alerted of the change when it happens. Powerful ways
+to leverage webhooks with Liveblocks include:
+
+- Storage Synchronization between room(s) and internal database
+- Alerting the client when a user leaves or enters a room
+- Notifying the client if max concurrency has been reached
+
+### Setup
+
+## Creating an endpoint
+
+Creating endpoints is simple, and is done from within the webhooks dashboard for
+your project.
+
+Before sending events, you should disable CSRF protection for the endpoint if it
+is enabled by default, and use the steps outlined in the Security Section to
+ensure the validity of the message.
+
+If your service needs custom headers to be set to work with load balancers or
+application servers, the Advanced section of the Endpoint configuration provides
+the option to configure these values, alongside custom rate limiting.
+
+<Will insert video here>
+
+## Subscribing to events
+
+Each endpoint you provide in the webhooks dashboard listens to all events by
+default but can be easily configured to only listen to a subset by updating the
+Message Filtering section during endpoint creation. The subscribed events can be
+edited later in the Endpoints Overview.
+
+<will show screenshot of where to edit the event >
+
+## Replaying events
+
+If your service is unreachable, retries will automatically be attempted. If your
+service incurs considerable downtime (over 10 hours), you can replay individual
+messages from the Endpoints portion of the dashboard by clicking the kebab menu
+on an individual message, or you can opt to bulk replay events by clicking the
+main menu and selecting "Replay Failed Messages." For a list of the specific
+retry intervals, see the
+[Svix Documentation on Retries](https://docs.svix.com/retries)
+
+<will video here>
+
+## Security
+
+Webhooks have the ability to generate security holes, but with Svix verification
+features this can be mitigated. When you verify your webhooks using Svix, you
+protect against man-in-the-middle, CSRF, and replay attacks. Because of this, the
+importance of verification cannot be understated and should be considered
+paramount in your integration. For a deeper dive into why you should verify
+webhooks, the
+[Svix Documentation](https://docs.svix.com/receiving/verifying-payloads/why) is
+a great resource.
+
+In order to validate a webhook with the Svix libraries run:
```

Comment:
```suggestion
In order to validate a webhook with the Svix libraries, run:
```

Felt weird without punctuation, but not sure my suggestion is good

Reply:
I think this part is the section where Flo was going to edit